linkerd/app/outbound/src/lib.rs

```diff
@@ -151,12 +152,18 @@ impl Outbound<()> {
         C::ResponseBody: Send + 'static,
         C::Future: Send,
     {
-        policy::Api::new(workload, limits, Duration::from_secs(10), client)
-            .into_watch(backoff)
-            .map_result(|response| match response {
-                Err(e) => Err(e.into()),
-                Ok(rsp) => Ok(rsp.into_inner()),
-            })
+        policy::Api::new(
+            workload,
+            limits,
+            Duration::from_secs(10),
+            export_hostname_labels,
+            client,
+        )
+        .into_watch(backoff)
+        .map_result(|response| match response {
```

Comment:
nit: we tend to use 'res' when handling a one off result (i.e. as compared to rsp for response).

```suggestion
        .map_result(|res| match res {
```